app/controllers/assignments_controller.rb

```diff
@@ -255,6 +253,8 @@ def show_Exam
 
   # setup graders
   def set_exam_graders
+    # FIXME: This is too complicated.
+
```

Comment:
He'd say, would that our Fundies 1 datatypes got that sophisticated!

(If you meant, why was my comment duplicated, it's because my justification for why this code is complicated is the same...but it is a different datatype, hence different code.)